psycopg2.sln

```diff
@@ -26,7 +25,7 @@ Global
 		$2.DirectoryNamespaceAssociation = None
 		$2.ResourceNamePolicy = FileName
 		$0.StandardHeader = $3
-		$3.Text = 
+		$3.Text =

```

Comment:
I guess this crap is autogenerated: not worth fighting against it.

Reply:
sln is an artifact from using old MD ide for C/C++. Imho can be removed.

Reply:
Good solution :)